app/src/main/java/com/example/returnpals/composetools/loginOptions.kt

```diff
@@ -79,4 +79,43 @@ class loginOptions {
                 }
             }
     }
+    @Composable
+    fun drawGuestLoginUI(userSignIn: () -> Unit, signin:() -> Unit, signup: () -> Unit) {
```

Comment:
Another code style thing that bugs me: I'd prefer we followed the naming conventions already established by jetpack compose. Imo, `drawGuestLoginUI` should be `GuestLoginUI`. When you call that `Test` and `Button` functions they aren't called `drawText` or `drawButton`. However, if this was Java or some other language I would completely agree with your naming style. Again, not a huge deal just something that irks me. I'm completely willing to look past it, I just wanted to bring it up.

Reply:
As far as I know jetpack compose follows this naming scheme because it tries to be declarative. So you declare that `Text` exists and then it does, rather than describing an action like draw like you would typically do in java such as `Text.draw()`

Reply:
Again, good info to share. Thanks!  I will change my code later to fit our group's coding style.  I will close this convo when we have a meeting scheduled to create a standard coding style.